inc/vcf/fixer.hpp

```diff
@@ -174,29 +223,272 @@ namespace ebi
         }
 
       protected:
+
+        /**
+         * puts the genotype as missing. if the error.cardinality is know, it uses the proper ploidy
+         * @param first iterator to the FORMAT column string
+         * @param last iterator past the last sample column
+         * @param error needed for the field (that must be "GT") the cardinality, and the line number
+         */
+        void fix_format_gt(std::vector<std::string>::iterator first,
+                           std::vector<std::string>::iterator last,
+                           SamplesFieldBodyError &error)
+        {
+            const std::string field_separator = ":";
+            size_t gt_column_index = 0;
+            // if the field is GT, the values must use "/", as in "./."
+            const std::string subfield_separator = "/";
+            const char empty_subfield = '.';
```

Comment:
our string_split allows several separators and and print_container uses strings for separators. We could use chars, but then we would need conversions to string later.
I would let this as-is.